### PR TITLE
explain makefile options

### PR DESCRIPTION
**Pull request by 1uka**
update readme to explain Makefile options, issue #349

### CODE REVIEW

**Comment by 1uka** [comment]
on it, forgot that it got updated in the meantime

**Comment by 1uka** [comment]
done, hope it is good

File: README.md [hunk 1]
```diff
@@ -25,6 +25,43 @@ cd poseidon
...
+make run_poseidon
+```
+This first builds poseidon, then runs it. After it finishes running, the container is removed.
+
+To populate the current volume with the contents of the containers' "poseidonWork/" directory, run:
+```
+make run_dev
+```
+
+To run poseidon with sh as entrypoing, run:
```
Comment (cglewis) [inline README.md]: typo: `entrypoint`

**Comment by cglewis** [comment]
almost there, thanks!

**Comment by 1uka** [comment]
fixed it, sorry if you waited had to go afk

**Pull request merged**